crates/ln-dlc-node/src/tests/multi_hop_payment.rs

```diff
@@ -17,21 +16,18 @@ async fn multi_hop_payment() {
     let payee = Node::start_test_app("payee").await.unwrap();
 
     payer.keep_connected(router.info).await.unwrap();
-    router.keep_connected(payee.info).await.unwrap();
+    payee.keep_connected(router.info).await.unwrap();
 
     payer.fund(Amount::from_sat(50_000)).await.unwrap();
-    router.fund(Amount::from_sat(50_000)).await.unwrap();
+    router.fund(Amount::from_sat(100_000)).await.unwrap();
 
-    payer.open_channel(router.info, 20_000, 0).await.unwrap();
-    router.open_channel(payee.info, 20_000, 0).await.unwrap();
+    router.open_channel(&payer, 20_000, 20_000).await.unwrap();
+    router.open_channel(&payee, 20_000, 20_000).await.unwrap();
 
     let payer_balance_before = payer.get_ldk_balance();
     let router_balance_before = router.get_ldk_balance();
     let payee_balance_before = payee.get_ldk_balance();
 
-    // Add 6 confirmations for the channels to be announced
-    mine(6).await;
```

Comment:
:wrench: We should remove the subsequent syncs and sleep since we aren't mining anything.

Reply:
the syncs are still needed to get every node up to date on the channels, but I removed the sleep.

https://github.com/get10101/10101/pull/169/commits/dc8c5e2b43923035927af3e2710f5e9e821052c9